.pre-commit-config.yaml

```diff
@@ -19,27 +17,18 @@ repos:
     hooks:
     -   id: black
         args: [--check]
```

Comment:
Shouldn't you add --diff here too?

Reply:
Can do - depends what we thing would be most helpful to the user. Personally I'd find it too verbose.

Reply:
OK! I'd be happy with less verbose indeed. It should just point us to where to run black.